src/pages/index.tsx

```diff
@@ -55,7 +57,7 @@ const HomePage: NextPageWithLayout = () => {
         <NotificationsAlert />
 
         <ComponentWrapperPage
-          src={components.wrapper}
+          src={sidebarLayoutHasInitialized ? components.gateway.home : components.wrapper}
```

Comment:
@calebjacob I'm not sure this is the best way to show this new home page for logged in users along with sidebar, so feel free to leave your thoughts about that